spec/javascripts/components/stories/history_spec.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import History, {
+  Changes,
+  Header,
+  getDate
+} from "../../../../app/assets/javascripts/components/stories/History";
+import HistoryFactory from "../../support/factories/historyFactory";
+
+describe("<History />", () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(<History history={HistoryFactory()} />);
```

Comment:
Can we use shallow instead of mount?

Reply:
We could do that but with `shallow` the subcomponents(`Header`, `Changes`) will not render. So i'll have to shallow each component separately and pass the right props, that's ok?